### ISSUE TITLE
PHP: Cannot start vagrant under windows 7 with Cygwin

### ISSUE THREAD

**Issue by mbunge** [hashicorp/otto#157]
Otto is not able to start PHP under Version 7

Appfile:

```hcl
application {
    name = "otto-php-example"
    type = "php"
}
```

CLI Output
```
$ otto dev
←[0m←[1m==> Creating local development environment with Vagrant if it doesn't exist...←[0m
←[0m    Raw Vagrant output will begin streaming in below. Otto does
    not create this output. It is mirrored directly from Vagrant
    while the development environment is being created.
    ←[0m
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'hashicorp/precise64' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'poweroff' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.
←[0m←[0m
←[0;31mError building dev environment: Error executing Vagrant: exit status 1

The error messages from Vagrant are usually very informative.
Please read it carefully and fix any issues it mentions. If
the message isn't clear, please report this to the Otto project.←[0m
```

**Comment by johnyesberg** [comment]
What happens if you start VirtualBox (to watch what happens) and in your command prompt, run "vagrant up"?

**Comment by mbunge** [comment]
I get the error which is also coming when Otto is executing `vagrant up`

```
==> default: Checking if box 'hashicorp/precise64' is up to date...
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 => 2222 (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'poweroff' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.
←[0m←[0m
←[0;31mError building dev environment: Error executing Vagrant: exit status 1

The error messages from Vagrant are usually very informative.
Please read it carefully and fix any issues it mentions. If
the message isn't clear, please report this to the Otto project.←[0m
```

**Comment by johnyesberg** [comment]
Does the box show any signs of starting up in the VirtualBox window?
I wonder if there is a problem with that particular box.
Does it make any difference if you delete the box and start again?

**Comment by mbunge** [comment]
This is fixed since 0.2
